cheddar/cheddar-tx/src/test/java/com/clicktravel/cheddar/infrastructure/persistence/database/tx/TransactionalDatabaseTemplateTest.java

```diff
@@ -264,9 +268,11 @@ public void shouldCommitCreate_withPersistenceExceptionHandler() throws Exceptio
     @Test
     public void shouldNotCommitCreate_withPersistenceExceptionHandlerThrowingException() throws Exception {
         // Given
-        final PersistenceExceptionHandler<PersistenceException> persistenceExceptionHandler = mock(PersistenceExceptionHandler.class);
+        final PersistenceExceptionHandler<PersistenceException> persistenceExceptionHandler = mock(
+                PersistenceExceptionHandler.class);
         final StubItem item = randomStubItem();
         final PersistenceException persistenceException = mock(PersistenceException.class);
+        when(persistenceExceptionHandler.getExceptionClass()).thenAnswer(new Returns(PersistenceException.class));
```

Comment:
No, it doesn't work because of wildcard type. I get a compilation error:
`The method thenReturn(Class<capture#7-of ? extends PersistenceException>) in the type OngoingStubbing<Class<capture#7-of ? extends PersistenceException>> is not applicable for the arguments (Class<PersistenceException>)`